src/main/java/com/softserveinc/dokazovi/aop/PostLogger.java

```diff
@@ -79,21 +78,6 @@ public Boolean updatePost(ProceedingJoinPoint proceedingJoinPoint) throws Throwa
         return joinPoint;
     }
 
-    @AfterReturning("execution(* com.softserveinc.dokazovi.service.impl.PostServiceImpl.removePostById("
-            + "com.softserveinc.dokazovi.security.UserPrincipal,"
-            + "Integer, boolean))")
-    public void deletePost(JoinPoint joinPoint) {
-        Object[] arguments = joinPoint.getArgs();
-        UserPrincipal userPrincipal = getArgumentFromArrayByClassType(arguments, UserPrincipal.class);
-        Integer postId = getArgumentFromArrayByClassType(arguments, Integer.class);
-        boolean flag = getArgumentFromArrayByClassType(arguments, Boolean.class);
-        if (!flag) {
-            return;
-        }
-        PostEntity postEntity = postRepository.getOne(postId);
-        makeEntryInLogs(postEntity.getTitle(), userPrincipal, "Матеріал видалено", null);
-    }
-
     private void makeEntryInLogs(String title, UserPrincipal userPrincipal, String changes, Integer postId) {
```

Comment:
Nit: you forgot to drop this.
Can and should be dropped, but if you don't have time, which is true as far as I am aware - feel free to do that in a separate follow-up PR.
Call LogService here when you get to it.